query/logicalplan/expr.go

```diff
@@ -99,8 +99,8 @@ func (e *BinaryExpr) Computed() bool {
 	return true
 }
 
-func (e *BinaryExpr) Alias(alias string) AliasExpr {
-	return AliasExpr{Expr: e, Alias: alias}
+func (e *BinaryExpr) Alias(alias string) *AliasExpr {
```

Comment:
Does this need to be a pointer?

Reply:
Yes, all the `AliasExpr` functions are pointer receivers so I believe it was a typo to not have it be one already.